pkg/apis/application/v1alpha1/openapi_generated.go

```diff
@@ -105,7 +105,6 @@ func schema_pkg_apis_application_v1alpha1_AWSAuthConfig(ref common.ReferenceCall
 				},
 			},
 		},
-		Dependencies: []string{},
```

Comment:
The codegen changes to this file are back. I have ensured I have the correct versions of `protobuf` and `swagger`. Is there anything else I should do to ensure this file is generated as intended?